packages/manager/cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts

```diff
@@ -386,3 +396,115 @@ describe('linode landing checks', () => {
+
+describe('linode landing checks for empty state', () => {
+  beforeEach(() => {
+    const mockAccountSettings = accountSettingsFactory.build({
+      managed: false,
+    });
+
+    cy.intercept('GET', apiMatcher('account/settings'), (req) => {
+      req.reply(mockAccountSettings);
+    }).as('getAccountSettings');
+    cy.intercept('GET', apiMatcher('profile')).as('getProfile');
+    cy.intercept('GET', apiMatcher('linode/instances/*'), (req) => {
+      req.reply(emptyLinodeData);
+    });
+    cy.visitWithLogin('/', { preferenceOverrides });
+    cy.wait('@getAccountSettings');
+    cy.url().should('endWith', routes.linodeLanding);
+  });
+
+  it('checks empty state on linode landing page', () => {
+    cy.get('div[data-qa-placeholder-container="resources-section"]').as(
+      'resourcesSection'
+    );
+    cy.get('@resourcesSection')
+      .get('h1[data-qa-header]')
+      .contains('Linodes')
+      .as('linodesHeader');
+
+    // Assert that fields with Linodes and Cloud-based virtual machines text are visible
+    cy.get('@linodesHeader').should('be.visible');
+    cy.get('@linodesHeader')
+      .next('h2')
+      .should('be.visible')
+      .should('have.text', 'Cloud-based virtual machines');
+
+    //Assert that recommended section is visible - Getting Started Guides, Deploy an App and Video Playlist
+    cy.get('@resourcesSection')
+      .contains('h2', 'Getting Started Guides')
+      .should('be.visible');
+    cy.get('@resourcesSection')
+      .contains('h2', 'Deploy an App')
+      .should('be.visible');
+    cy.get('@resourcesSection')
+      .contains('h2', 'Video Playlist')
+      .should('be.visible');
+
+    // Assert that Create Linode button is visible and enabled
+    ui.button
+      .findByTitle('Create Linode')
+      .should('be.visible')
+      .and('be.enabled');
+
+    // Assert that List of Liondes table does not exist
+    cy.get('table[aria-label="List of Linodes"]').should('not.exist');
+
+    // Assert that Docs link does not exist
+    cy.get(
+      'a[aria-label="Docs - link opens in a new tab"][data-testid="external-link"]'
+    ).should('not.exist');
+
+    // Assert that Download CSV button does not exist
+    cy.get('span[data-testid="loadingIcon"]')
+      .contains('Download CSV')
+      .should('not.exist');
+  });
+});
+
+describe('linode landing checks for restricted user', () => {
+  beforeEach(() => {
+    const mockProfile = profileFactory.build({
+      username: randomLabel(),
+      restricted: true,
+    });
+
+    const mockUser = accountUserFactory.build({
+      username: mockProfile.username,
+      restricted: true,
+      user_type: 'default',
+    });
+
+    const mockGrants = grantsFactory.build({
+      global: {
+        add_linodes: false,
+      },
+    });
+
+    mockGetProfile(mockProfile);
+    mockGetProfileGrants(mockGrants);
+    mockGetUser(mockUser);
+    cy.intercept('GET', apiMatcher('linode/instances/*'), (req) => {
+      req.reply(emptyLinodeData);
+    });
```

Comment:
```suggestion
    mockGetLinodes([]);
```
Same suggestion as my other comment!

Reply:
Made changes in this commit https://github.com/linode/manager/pull/10882/commits/dbc04a5811cc5c60a8e16a7c8c058708ec04025f